tests/upgrade/test_crossgrade.sh

```diff
@@ -116,12 +116,16 @@ installIstioSystemAtVersionHelmTemplate() {
         auth_opts="--set global.mtls.enabled=true --set global.controlPlaneSecurityEnabled=true "
     fi
     release_path="${3}"/install/kubernetes/helm/istio
-    helm init --client-only
-    helm dep update "${release_path}"
+    if [[ "${release_path}" = *"1.1"* ]]; then
```

Comment:
perhaps something like "${release_path}" != *"1.0"* instead to be more future proof?

Reply:
You're a real optimist if you think this will be future proof :-) I think each release will be needing a tweak.

Reply:
I am hoping to have the same file across branches. Otherwise it is a nightmare to reconcile all these after every merges.

We are already having problem between release-1.1 and master and I don't want to make it worse.